src/test/resources/features/AmendCYAPage.feature

```diff
@@ -48,27 +46,20 @@ Feature: Amend case CYA Page
     #unselecting further information
     And I choose both send more supporting documents and give further information
     When I click on the "Continue" button
-    Then I am on "Upload a file - Apply for repayment of import duty and import VAT - GOV.UK" page
+    Then I am on "Upload your additional documents - Apply for repayment of import duty and import VAT - GOV.UK" page
     When I click on Choose file button and add the "first" file
-    When I click on the "Continue" button
-    Then I am on "You have uploaded 1 file - Apply for repayment of import duty and import VAT - GOV.UK" page
-    Then I should see first uploaded doc "JPEGImage.jpg" on upload review page
-    Then I select "<YorNuploadFile>" to uploading another file
-    When I click on the "Continue" button
+    And I wait for the file to be uploaded
+    Then I should see first uploaded doc "JPEGImage.jpg" on upload page
+    When I click the file continue button
     Then I am on "Check your answers before sending your information - Apply for repayment of import duty and import VAT - GOV.UK" page
     Then I verify correct responses is displayed for Amend case journey "<changeReferenceNo>", "<changeRespType>", "<docsUploaded>"
 
     #change Documents
     When I click on change option on the amend review page for documents uploaded
-    Then I am on "You have uploaded 1 file - Apply for repayment of import duty and import VAT - GOV.UK" page
-    Then I select "<YorNuploadFiles>" to uploading another file
-    When I click on the "Continue" button
-    Then I am on "Upload another file - Apply for repayment of import duty and import VAT - GOV.UK" page
+    Then I am on "Upload your additional documents - Apply for repayment of import duty and import VAT - GOV.UK" page
     When I click on Choose file button and add the "second" file
-    When I click on the "Continue" button
-    Then I am on "You have uploaded 2 files - Apply for repayment of import duty and import VAT - GOV.UK" page
-    Then I select "<YorNuploadFile>" to uploading another file
-    When I click on the "Continue" button
+    And I wait for the file to be uploaded
```

Comment:
AMBER: Should we not check here that the second file actually uploaded and is visible and confirm the first file has not been removed

```
Then I should see first uploaded doc "JPEGImage.jpg" on upload page
Then I should see second uploaded doc "VA Plan v0.3.xlsx" on upload page
```

Reply:
Done